frontend/index.html

```diff
@@ -12,72 +12,74 @@
 </head>
 
 <body>
-<!-- NavBar -->
-  <nav class="home-mobile-nav">
-    <div class="nav-wrapper">
-      <ul class="side-nav" id="mobile-demo">
-        <li><a href="index.html" class="mobile-nav-logo"><img src="images/navbar-logo.png" alt="logo"></a></li>
-
-        <li><div class="divider"></div></li>
-
-       <li>
-         <a href="#!" class="waves-effect waves-light btn-large">Start Learning</a>
-       </li>
-        <li><a href="index.html"><i class="material-icons">home</i>Home</a></li>
-        <li><a href="explorer.html"><i class="material-icons">library_books</i>Topic Library</a></li>
-        <li><a href="certification.html"><i class="material-icons">school</i>Certification</a></li>
-
-        <li><div class="divider"></div></li>
-
-        <li><a href="#!"><i class="material-icons">work</i>Tools</a></li>
-        <li><a href="#!"><i class="material-icons">insert_invitation</i>Events</a></li>
-        <li><a href="partners.html"><i class="material-icons">people</i>Partners</a></li>
-        <li><a href="help.html"><i class="material-icons">help</i>Help & FAQs</a></li>
-        <li><a href="feedback.html"><i class="material-icons">chat_bubble</i>Feedback</a></li>
-
-        <li><div class="divider"></div></li>
-
-        <li><a href="#!"><i class="material-icons">thumb_up</i>Blog</a></li>
-        <li><a href="#!" class="waves-effect waves-light btn-large">Sign in</a></li>
-      </ul>
-    </div>
-  </nav>
-
-  <ul id='dropdown1' class='dropdown-content'>
-    <li><a href="tools-page.html">Tools</a></li>
-    <li class="divider"></li>
-    <li><a href="help.html">Help & FAQs</a></li>
-    <li><a href="feedback.html">Feedback</a></li>
-    <li><a href="#!">Blog</a></li>
-    <li class="divider"></li>
-    <li><a href="#!">My Profile</a></li>
-    <li><a href="#!">Sign out</a></li>
-  </ul>
-
-  <div class="navbar-fixed">
-    <nav class="nav-extended">
-      <div class="nav-wrapper pad-5">
-        <a href="#!home" class="brand-logo"><img src="images/navbar-logo.png" alt="logo"></a>
-        <div class="navbar-links">
-          <ul class="left hide-on-med-and-down">
-            <li><a href="index.html">Home</a></li>
-            <li><a href="explorer.html">Topic Library</a></li>
-            <li><a href="certification.html">Certification</a></li>
-            <li><a href="#!">Events</a></li>
-
-            <!-- Dropdown Trigger -->
-            <li><a class="dropdown-button" href="#!" data-activates="dropdown1">More<i class="material-icons right">arrow_drop_down</i></a></li>
-          </ul>
-
-          <ul class="right hide-on-med-and-down sign-in-nav">
-            <li><a class="sign-in-link" href="#!" target="_blank">SIGN IN</a></li>
-            <li><a href="#!" target="_blank" class="waves-effect waves-light btn">Start Learning</a></li>
-          </ul>
-      </div>
+
```

Comment:
@Sharonsyra Did we just introduce a blank line that then triggered all these additions?

Reply:
I was just spacing to separate different components